schema_source/definitions.json

```diff
@@ -74,6 +74,14 @@
           "type": "string",
           "description": "The name extracted from the user JWT claim"
         },
+        "kind": {
+          "type": "string",
+          "description": "The kind of client. Can be Client/Leafnode/Router/Gateway/JetStream/Account/System"
```

Comment:
schemas do support enums which could be handy, but probably ok as is since most cases will just print whatever is given i guess

Reply:
Does this really update all the API audits and everything else too?

Reply:
After having seen all the changes I went back and made sure to set kind/client type whenever client info is set.